### ISSUE TITLE
Implement Age 11 cards for Unseen Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#1084]
This issue will track the implementation of all Age 11 cards for the Unseen expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

- [x] 585 : Astrobiology
- [x] 586 : Quantum Computers
- [x] 587 : Cloaking
- [x] 588 : Dark Web
- [x] 589 : Green Hydrogen
- [x] 590 : Mystery Box
- [x] 591 : Escape Room
- [x] 592 : Fashion Mask
- [x] 593 : Holography
- [x] 594 : Metaverse
